### PR TITLE
Add Wix Installer project

### PR DESCRIPTION
**Pull request by gayashanbc**
$subject

### CODE REVIEW

File: io.asgardio.dotnet.oidc.sdk.sample.setup/OAuthConfigDialog.wxs [hunk 1]
```diff
@@ -0,0 +1,62 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<!--
+  ~ Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Comment (darshanasbg) [inline io.asgardio.dotnet.oidc.sdk.sample.setup/OAuthConfigDialog.wxs:3]: should be 2020?

File: io.asgardio.dotnet.oidc.sdk.sample.setup/io.asgardio.dotnet.oidc.sdk.sample.setup.wixproj [hunk 2]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (darshanasbg) [inline io.asgardio.dotnet.oidc.sdk.sample.setup/io.asgardio.dotnet.oidc.sdk.sample.setup.wixproj:1]: licenses?

File: io.asgardio.dotnet.oidc.sdk.sample.setup/io.asgardio.dotnet.oidc.sdk.sample.setup.wixproj [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    <Compile Include="OAuthConfigDialog.wxs" />
+  </ItemGroup>
+  <Import Project="$(WixTargetsPath)" Condition=" '$(WixTargetsPath)' != '' " />
+  <Import Project="$(MSBuildExtensionsPath32)\Microsoft\WiX\v3.x\Wix.targets" Condition=" '$(WixTargetsPath)' == '' AND Exists('$(MSBuildExtensionsPath32)\Microsoft\WiX\v3.x\Wix.targets') " />
+  <Target Name="EnsureWixToolsetInstalled" Condition=" '$(WixTargetsImported)' != 'true' ">
+    <Error Text="The WiX Toolset v3.11 (or newer) build tools must be installed to build this project. To download the WiX Toolset, see http://wixtoolset.org/releases/" />
+  </Target>
+  <!--
+	To modify your build process, add your task inside one of the targets below and uncomment it.
+	Other similar extension points exist, see Wix.targets.
```
Comment (darshanasbg) [inline io.asgardio.dotnet.oidc.sdk.sample.setup/io.asgardio.dotnet.oidc.sdk.sample.setup.wixproj:35]: Remove this part.

File: io.asgardio.dotnet.oidc.sdk.sample.setup/OAuthConfigDialog.wxs [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+  <Fragment>
+    <UI>
+      <Dialog Id="OAuthConfigDialog" Width="370" Height="427" Title="[ProductName] [Setup]" NoMinimize="yes">
+        <Control Id="ClientIdLabel" Type="Text" X="45" Y="73" Width="100" Height="15" TabSkip="no" Text="&amp;Client ID:" />
+        <Control Id="ClientIdEdit" Type="Edit" X="45" Y="85" Width="265" Height="18" Property="CLIENT_ID" Text="{80}" />
+        <Control Id="ClientSecretLabel" Type="Text" X="45" Y="110" Width="100" Height="15" TabSkip="no" Text="&amp;Client Secret:" />
+        <Control Id="ClientSecretEdit" Type="Edit" X="45" Y="122" Width="265" Height="18" Property="CLIENT_SECRET" Text="{80}" />
+        <Control Id="AuthorizationEndpointLabel" Type="Text" X="45" Y="147" Width="100" Height="15" TabSkip="no" Text="&amp;Authorization Endpoint:" />
+        <Control Id="AuthorizationEndpointEdit" Type="Edit" X="45" Y="159" Width="265" Height="18" Property="AUTHORIZATION_ENDPOINT" Text="{80}" />
+        <Control Id="TokenEndpointLabel" Type="Text" X="45" Y="184" Width="100" Height="15" TabSkip="no" Text="&amp;Token Endpoint:" />
```
Comment (darshanasbg) [inline io.asgardio.dotnet.oidc.sdk.sample.setup/OAuthConfigDialog.wxs:30]: Don't we have a way to reuse these styling?

File: io.asgardio.dotnet.oidc.sdk.sample.setup/OAuthConfigDialog.wxs (hunk 1 above)
Comment (gayashanbc) [reply io.asgardio.dotnet.oidc.sdk.sample.setup/OAuthConfigDialog.wxs]: Resolved in https://github.com/asgardio/asgardio-dotnet-oidc-sdk/pull/6/commits/0385a6d8936ff3b7c647d680cf0b9a29f3ebc140

File: io.asgardio.dotnet.oidc.sdk.sample.setup/io.asgardio.dotnet.oidc.sdk.sample.setup.wixproj (hunk 2 above)
Comment (gayashanbc) [reply io.asgardio.dotnet.oidc.sdk.sample.setup/io.asgardio.dotnet.oidc.sdk.sample.setup.wixproj:1]: Resolved in https://github.com/asgardio/asgardio-dotnet-oidc-sdk/pull/6/commits/0385a6d8936ff3b7c647d680cf0b9a29f3ebc140

File: io.asgardio.dotnet.oidc.sdk.sample.setup/io.asgardio.dotnet.oidc.sdk.sample.setup.wixproj (hunk 3 above)
Comment (gayashanbc) [reply io.asgardio.dotnet.oidc.sdk.sample.setup/io.asgardio.dotnet.oidc.sdk.sample.setup.wixproj]: Resolved in https://github.com/asgardio/asgardio-dotnet-oidc-sdk/pull/6/commits/0385a6d8936ff3b7c647d680cf0b9a29f3ebc140

File: io.asgardio.dotnet.oidc.sdk.sample.setup/OAuthConfigDialog.wxs (hunk 4 above)
Comment (gayashanbc) [reply io.asgardio.dotnet.oidc.sdk.sample.setup/OAuthConfigDialog.wxs:30]: no, we have to specify them explicitly

Comment (darshanasbg) [reply io.asgardio.dotnet.oidc.sdk.sample.setup/OAuthConfigDialog.wxs:30]: ack

**Review by darshanasbg — APPROVED** [review]

**Pull request merged**